### DISCUSSION TITLE
Generally cryptic error messages when using Prisma

### DISCUSSION THREAD

**Discussion by elijahharry** [planetscale/discussion#224] [Q&A] [answered]
The error messages lack the field name, making them relatively difficult to troubleshoot. Here's an example of one I'm currently dealing with:
```
Error: 
Invalid `prisma.comment.create()` invocation:
  Null constraint violation on the (not available)
```
'Not available' is a recurring theme in the messages I'm receiving. In the past, it turned out the lack of `onUpdate: NoAction` in the relation was causing it, i.e. this was the fix:
`organization   Organization @relation(fields: [organizationId], references: [id], onUpdate: NoAction)`
But there's no real way to even know where you should be directing your attention with the error messages received.

**Comment by janpio** [comment, marked as answer]
PlanetScale and Vitess unfortunately return the affected fields slightly different than MySQL, which causes Prisma to not be able to parse the message.
